test/000_fullchain-boost.test.ts

```diff
@@ -157,17 +162,11 @@ describe('IexecPocoBoostDelegate (IT)', function () {
             await signOrders(domain, orders, ordersActors);
             const dealId = getDealId(domain, requestOrder, taskIndex);
             const startTime = await setNextBlockTimestamp();
-            const matchOrdersArgs = [
-                appOrder,
-                datasetOrder,
-                workerpoolOrder,
-                requestOrder,
-            ] as Orders;
 
             expect(
-                await iexecPocoBoostInstance.callStatic.matchOrdersBoost(...matchOrdersArgs),
+                await iexecPocoBoostInstance.callStatic.matchOrdersBoost(...orders.toArray()),
```

Comment:
I liked the fact this was names matchOrdersArgs

Reply:
The variable was not very useful IMO.